appveyor.yml

```diff
@@ -39,7 +39,10 @@ environment:
 build_script:
   - cmake "-G%GENERATOR%" -H. -Bbuild
   - cmake --build build --config "%CONFIG%"
+  # Intentionally failing the build to check the test
+  #- cmake --build build --config "%CONFIG%" binding-functions

```

Comment:
Should this still be here? Does leaving it commented out imply that the test should fail but doesn't?

Reply:
The test should fail, but it just emits an error message and marks it as succeeded anyways. I'd like to take a look at that tomorrow and see if I can get it working before merging.

Reply:
I updated the PR so Appveyor also tests the binding-functions target.